autobahn/twisted/websocket.py

```diff
@@ -515,17 +520,17 @@ def connectWS(factory, contextFactory=None, timeout=30, bindAddress=None):
     else:
         from twisted.internet import reactor
 
+    if factory.isSecure:
+        if contextFactory is None:
+            # create default client SSL context factory when none given
+            from twisted.internet import ssl
+            contextFactory = ssl.ClientContextFactory()
+
```

Comment:
Shouldn't this be ``optionsForClientTLS(factory.host)`` to get browser-like verification?